plugin-packs/postcss-preset-env/src/index.js

```diff
@@ -97,7 +97,7 @@ const plugin = opts => {
 				log(`  ${feature.id} with stage ${feature.stage} has been disabled`);
 			}
 		} else if (!isAllowedByType) {
-			log(`  ${feature.id} has been disabled by "disableClientSidePolyfills".`);
+			log(`  ${feature.id} has been disabled by "enableClientSidePolyfills: false".`);
```

Comment:
After switching this line became a bit weird : 
`has been disabled by "enableClientSidePolyfills"`

so I added the `: false` bit.